Configuration/StandardSequences/python/DigiToRaw_cff.py

```diff
@@ -49,7 +49,8 @@
 
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
 from Configuration.Eras.Modifier_phase2_GEM_cff import phase2_GEM
-(run3_GEM & ~phase2_GEM).toReplaceWith(DigiToRawTask, _gem_DigiToRawTask)
+run3_GEM.toReplaceWith(DigiToRawTask, _gem_DigiToRawTask)
+phase2_GEM.toReplaceWith(DigiToRawTask, _gem_DigiToRawTask)
```

Comment:
To have this `toReplaceWith` in both run3 and phase2 seems to be a duplication in the current way we are doing the `modifier`. Currently, `phase2` modifier uses `run3` as a basis. However, it is not harm to be in this way, and it is maybe a good way if we clean up the modifier in future.